accelerometer/device.py

```diff
@@ -9,6 +9,54 @@
 import struct
 from subprocess import call
 import sys
+import jpype
+import atexit
+import shutil
+from tempfile import mkstemp
+import re
+
+
+
+def parse(inputFile, outputFile, **kwargs):
+
+    if inputFile.endswith('.gz'):  # Decompress to temporary file
+        ext = re.search(r'(\.\w+)\.gz', inputFile).group(1)
```

Comment:
Is this really needed?
The previous Java implementations were able to read the compressed .gz files ... an advantage of this is a reduced storage footprint (rather than having to create a temporary uncompressed file)

Reply:
Thanks Aiden. It appears that we only got the functionality only for Axivity so far, so I went for the easiest approach. But yes I agree with you and will look into implementing .gz reads in Actigraph and GENEA.  

![Captura de pantalla de 2021-02-13 08-06-22](https://user-images.githubusercontent.com/5551642/107848629-b52bb680-6dd3-11eb-9cc2-b9c3fb2dc44f.png)

Reply:
Thanks Aiden. It appears that we only got the [functionality only for Axivity so far](https://github.com/activityMonitoring/biobankAccelerometerAnalysis/blob/master/java/AccelerometerParser.java#L214-L230), so I went for the easiest approach. But yes I agree with you and will look into implementing .gz direct reads in Actigraph and GENEA.